spec/controllers/picture_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "spec_helper"
+
+describe PictureController do
+  let(:picture_content) { "BINARY IMAGE CONTENT" }
+  let(:service_picture) { FactoryGirl.create(:picture, :extension => "jpg") }
```

Comment:
Might be cleaner to take the word "service" out, since the picture controller can field anything in the pictures table.